api/src/main/java/com/epam/pipeline/manager/region/CloudRegionApiService.java

```diff
@@ -39,6 +40,10 @@
         return cloudRegionManager.loadAll();
     }
 
+    public List<CloudRegionInfo> loadAllRegionsInfo() {
```

Comment:
We still need `@PreAuthorize` annotation here `hasRole('ADMIN') OR hasRole('ROLE_USER')`